app/views/groups/show.html.erb

```diff
@@ -13,6 +13,9 @@
           <% nb.with_heading(text: t("groups.show.trial_banner.heading"), tag: "h3") %>
           <%= t("groups.show.trial_banner.upgrade.body_html") %>
           <%= govuk_link_to t("groups.show.trial_banner.upgrade.link"), upgrade_group_path(@group) %>
+        <% elsif @current_user.super_admin? %>
+          <% nb.with_heading(text: t("groups.index.no_mou_banner.heading"), tag: "h3") %>
+          <%= t("groups.index.no_mou_banner.body_html", contact_link: contact_link) %>
```

Comment:
Why do only super admins see the paragraph about the MOU? Should all user types see this if there is no MOU signed?

Reply:
Super admins are the only ones who will need to be shown that they can't upgrade the group solely because the organisation doesn't have an MOU. 

The prior `elsif` checks to see if the user can upgrade the group, but that will be false if there's no MOU - which means we need to still catch super admins here, as there can't be any organisation admins if there's no MOU.

We were getting a bit concerned with the readability for this particular if-chain...

Reply:
Aha yeh ok that makes sense! Didn't consider the fact that you can't have org admins without an MOU. Yeh this if chain is getting quite difficult to follow!

The thing that's slightly confusing is that on the Figma there are 2 different notification banners. There's one for "If their org has no MOU or org admin yet", and one for "If their org has no org admin". Are we definitely using the right one for group admins?

Reply:
If we should always show the MOU version of the banner if there is no MOU signed, would it be clearer to have a check in the first "if" case that an MOU is signed and if not use the `no_mou_banner` content? Would that make this easier to follow?